fullrt/dht.go

```diff
@@ -596,27 +596,6 @@ func (dht *FullRT) searchValueQuorum(ctx context.Context, key string, valCh <-ch
 		})
 }
 
-// GetValues gets nvals values corresponding to the given key.
-func (dht *FullRT) GetValues(ctx context.Context, key string, nvals int) (_ []RecvdVal, err error) {
```

Comment:
Killed because it was killed in the IpfsDHT implementation and generally isn't very useful #728